modules/nw-ovn-kubernetes-ansible-migration.adoc

```diff
@@ -0,0 +1,163 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc
+// * networking/ovn_kubernetee_network_provider/rollback-to-openshift-sdn.adoc
+
+ifeval::["{context}" == "rollback-to-openshift-sdn"]
+:rollback:
+endif::[]
+
+:_mod-docs-content-type: PROCEDURE
+ifndef::rollback[]
+[id="nw-ovn-kubernetes-ansible-migration-about_{context}"]
+= Using an Ansible playbook to migrate to the OVN-Kubernetes network plugin
+
+As a cluster administrator, you can use an Ansible collection, `network.offline_migration_sdn_to_ovnk`, to migrate from the OpenShift SDN Container Network Interface (CNI) network plugin to the OVN-Kubernetes plugin for your cluster. The Ansible collection includes the following playbooks:
+
+* `playbooks/migration-playbook.yml`:: This playbook migrates from the OpenShift SDN CNI plugin to the OVN-Kubernetes plugin.
+* `playbooks/rollback-playbook.yml`:: This playbook rolls back from the OVN-Kubernetes plugin to the OpenShift SDN CNI plugin.
```

Comment:
`playbooks/playbook-migration.yml`: consists of playbooks to be executed in a sequence to perform required steps for migration.
`playbooks/playbook-rollback.yml`: consists of playbooks to be executed in a sequence to perform required steps for rollback.

Reply:
:heavy_check_mark: